### PR TITLE
feat: MoD TC starting period

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

File: src/views/TradingCompetition/helpers.ts [hunk 1]
```diff
@@ -87,6 +87,32 @@ export const useCompetitionRewards = ({
...
+  userDarRewards,
+}: {
+  userCakeRewards: ReactText
+  userDarRewards: ReactText
+}) => {
+  const darPriceBUSD = useBUSDPrice(tokens.dar)
+  const cakeAsBigNumber = new BigNumber(userCakeRewards as string)
+  const darAsBigNumber = new BigNumber(userDarRewards as string)
+  const cakeBalance = getBalanceNumber(cakeAsBigNumber)
+  const darBalance = getBalanceNumber(darAsBigNumber)
```
Comment (chef-jojo) [inline src/views/TradingCompetition/helpers.ts:101]: 6

File: src/views/TradingCompetition/helpers.ts [hunk 2]
```diff
@@ -87,6 +87,32 @@ export const useCompetitionRewards = ({
...
+  const cakePriceBusd = useCakeBusdPrice()
+
+  const dollarValueOfTokensReward =
+    cakePriceBusd && darPriceBUSD
+      ? multiplyPriceByAmount(cakePriceBusd, cakeBalance) + multiplyPriceByAmount(darPriceBUSD, darBalance, 8)
```
Comment (chef-jojo) [inline src/views/TradingCompetition/helpers.ts:106]: 6

File: src/views/TradingCompetition/helpers.ts (hunk 1 above)
Comment (Chef-Yogi) [reply src/views/TradingCompetition/helpers.ts]: fixed

File: src/views/TradingCompetition/helpers.ts (hunk 2 above)
Comment (Chef-Yogi) [reply src/views/TradingCompetition/helpers.ts]: fixed

File: src/views/TradingCompetition/components/ClaimModal/index.tsx [hunk 3]
```diff
@@ -29,11 +28,11 @@ const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, use
   const { fetchWithCatchTxError, loading: isConfirming } = useCatchTxError()
   const { t } = useTranslation()
 
-  const { userRewardGroup, userCakeRewards, userMoboxRewards, userPointReward, canClaimMysteryBox, canClaimNFT } =
+  const { userRewardGroup, userCakeRewards, userDarRewards, userPointReward, canClaimMysteryBox, canClaimNFT } =
```
Comment (memoyil) [inline src/views/TradingCompetition/components/ClaimModal/index.tsx:31]: Eslint

File: src/views/TradingCompetition/components/ClaimModal/index.tsx [hunk 4]
```diff
@@ -78,14 +77,14 @@ const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, use
             <Text mt="8px">{t('Collectible NFT')}</Text>
           </Flex>
         ) : null}
-        {canClaimMysteryBox ? (
+        {/* {canClaimMysteryBox ? (
```
Comment (memoyil) [inline src/views/TradingCompetition/components/ClaimModal/index.tsx:80]: Can be removed

**Comment by memoyil** [comment]
LGTM 👍

File: src/views/TradingCompetition/types.ts [hunk 5]
```diff
@@ -87,28 +87,38 @@ export interface UserRewardsProps {
...
-    volume?: number
-    // eslint-disable-next-line camelcase
-    next_rank?: number
+  userLeaderboardInformation?: UserLeaderboardSharedInformation & {
     moboxVolumeRank?: string
     moboxVolume?: string
   }
 }
+
+export interface MoDYourScoreProps extends CompetitionProps {
```
Comment (memoyil) [inline src/views/TradingCompetition/types.ts:118]: Can you move this inside to ModYourScore component?

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**